src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -1235,11 +1235,6 @@ internal virtual XmlDictionaryString TopLevelElementNamespace
                 set { _ns = value; }
             }
 
-            internal virtual bool CanContainReferences
```

Comment:
I don't think we should remove this property. Some classes, e.g. `EnumDataContract`, have methods overriding this.